src/commands/sourcemaps/README.md

```diff
@@ -69,6 +69,14 @@ The value can be overriden with `--repository-url`.
 Example: With a remote `git@github.com:Datadog/example.git`, links pointing to `https://github.com/Datadog/example` are generated.
 This behavior can be overriden with links to `https://gitlab.com/Datadog/example` with the flag `--repository-url=https://gitlab.com/Datadog/example`.
 
+#### Setting the project path
```

Comment:
Maybe we can add a link to this section in the description of the flag above?

Reply:
Just added one 👍